modules/monitoring-querying-metrics-for-all-projects-with-mon-dashboard.adoc

```diff
@@ -4,8 +4,11 @@
 // * virt/support/virt-prometheus-queries.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
-= Querying metrics for all projects as a cluster administrator
+[id="querying-metrics-for-all-projects-with-mon-dashboard_{context}"]
+// The following section will be included in the administrator section, hence there is no need to include "administrator" in the title
+= Querying metrics for all projects with the {product-title} web console
+
+The {product-title} metrics query browser enables you to run Prometheus Query Language (PromQL) queries to examine metrics visualized on a plot. This functionality provides information about the state of a cluster and any user-defined workloads that you are monitoring.
```

Comment:
suggestion to avoid light anthropomorphism here:
```suggestion
You can use the {product-title} metrics query browser to run Prometheus Query Language (PromQL) queries to examine metrics visualized on a plot. This functionality provides information about the state of a cluster and any user-defined workloads that you are monitoring.
```